wom/src/main/scala/wom/callable/TaskDefinition.scala

```diff
@@ -1,13 +1,12 @@
 package wom.callable
```

Comment:
This whole class is a big bag of yuck, fortunately I was able to comment out all the really bad stuff for now

Reply:
Oh yeah this was a mess waiting to be cleaned up. I'd even say "comment out for good" and just remove it (eg there's no way WOM is going to want to be dealing with lookup functions)